llvm/include/llvm/Object/ELF.h

```diff
@@ -425,6 +425,8 @@ class ELFFile {
       std::function<Expected<bool>(const Elf_Shdr &)> IsMatch) const;
 
   void createFakeSections();
+
+  static bool isSectionInSegment(const Elf_Phdr &Phdr, const Elf_Shdr &Sec);
```

Comment:
Minor point: I'm not sure what the benefit is of making this a static function versus a non-member function, given it takes Elf_* types, so is clearly ELF-related.